sites/svelte.dev/content/blog/2023-05-01-whats-new-in-svelte-may-2023.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "What's new in Svelte: May 2023"
+description: "TBD"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+TBD... Svelte Hackathon winner goes here too.
+
```

Comment:
ESLint PR is merged. So I wrote some sentences.
I who is shameless put this on the top of the article, but is this the right place?
And I'm happy if someone modifies my English🙏

```suggestion
## New official ESLint plugin is landed 🚀

We're happy to announce that we have a new official [ESLint plugin](https://github.com/sveltejs/eslint-plugin-svelte)🎉 

Our existing ESLint plugin [eslint-plugin-svelte3](https://github.com/sveltejs/eslint-plugin-svelte3) has supported Svelte's development for a long time.
However, it does not handle the AST of the template well, which results in false positives/negatives and makes it very difficult to create custom ESLint rules. To solve this issue, [ota-meshi](https://github.com/ota-meshi) developed [svelte-eslint-parser](https://github.com/sveltejs/svelte-eslint-parser) and [eslint-plugin-svelte](https://github.com/sveltejs/eslint-plugin-svelte) which has proven to be more reliable and successful. The Svelte team therefore decided to switch the official plugin to [eslint-plugin-svelte](https://github.com/sveltejs/eslint-plugin-svelte).

Please try the new plugin! And we are waiting for your feedback!
```